File: samples/CS/16-RecentItems/Form1.cs
using System;
using System.Windows.Forms;

using WinForms.Ribbon;
using System.Collections.Generic;

namespace _16_RecentItems
{
    public partial class Form1 : Form
    {
        private RibbonItems _ribbonItems;

        public Form1()
        {
            InitializeComponent();
            _ribbonItems = new RibbonItems(_ribbon);
            _ribbonItems.Init();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            _ribbonItems.Load();
        }
    }
}

File: samples/VB/10-CheckBox/Form1.vb
Imports WinForms.Ribbon

Namespace _10_CheckBox

	Partial Public Class Form1
		Inherits Form
		Private _ribbonItems As RibbonItems
		Private _toggleButton As RibbonToggleButton
		Private _checkBox As RibbonCheckBox

		Public Sub New()
			InitializeComponent()
			_ribbonItems = New RibbonItems(_ribbon)
			_ribbonItems.Init()
		End Sub
	End Class
End Namespace
